src/containers/Admin/styles.scss

```diff
@@ -74,7 +74,7 @@
 
 .eds-tab-panel:focus,
 .eds-tab:focus {
-    outline: none;
+    outline: none !important;
```

Comment:
Hm, me bør egentlig ikkje fjerne fokusmarkering, for då vil det bli vanskelig å sjå kva som er markert.